### ISSUE TITLE
[Suggestion] Use a more robust download solution

### ISSUE THREAD

**Issue by etherealxx** [Crozzers/ffmpeg-installer#2]
This script is an amazing time saver, thanks for making this. However, i have some suggestion.

- The manual download method is hella slow. Could you use other option like wget/gdown/etc.?
- Please allow me to also use lowercase 'y' here😁
![image](https://github.com/Crozzers/ffmpeg-installer/assets/64251396/9ce63eb7-d158-42ea-9b4f-78fe5025bde3)

**Comment by Crozzers** [comment]
Hi, thanks for the suggestions!

I've slapped a commit on to allow lowercase 'y's for the PATH modification step. I'll look into the download speed suggestion and test it at some point in the near future

**Comment by Crozzers** [comment]
I've pushed some commits that should let you set which program will download the file. Included options are powershell's `Invoke-WebRequest`, `wget` and `curl`. I've updated the readme with the [new CLI args](https://github.com/Crozzers/ffmpeg-installer#Usage).

Wget will require you to have installed the "real" wget, since Windows doesn't ship that by default.

From my testing, wget and curl were the fastest, with `Invoke-WebRequest` being the slowest of them all.
